src/OpenApi/gen/XmlComments/MemberKey.cs

```diff
@@ -14,14 +14,19 @@ internal sealed record MemberKey(
     MemberType MemberKind,
     string? Name,
     string? ReturnType,
-    string[]? Parameters) : IEquatable<MemberKey>
+    List<string>? Parameters) : IEquatable<MemberKey>
 {
-    private static readonly SymbolDisplayFormat _typeKeyFormat = new(
+    private static readonly SymbolDisplayFormat _withTypeParametersFormat = new(
         globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Included,
         typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
         genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters);
 
-    public static MemberKey FromMethodSymbol(IMethodSymbol method, Compilation compilation)
+    private static readonly SymbolDisplayFormat _sansTypeParametersFormat = new(
+        globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Included,
+        typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
+        genericsOptions: SymbolDisplayGenericsOptions.None);
+
+    public static MemberKey? FromMethodSymbol(IMethodSymbol method, Compilation compilation)
```

Comment:
We'll return null here if we can't emit a member key. Based [on the code here](https://github.com/dotnet/aspnetcore/blob/afd6f4a27a271d6e746150fabdd358ab746c8720/src/OpenApi/gen/XmlCommentGenerator.Parser.cs#L112) this causes the comment associated with this type parameter to not be emitted into source code.